### ISSUE TITLE
update the DLO prefix for all artifacts in this repo

### ISSUE THREAD

**Issue by turbomam** [turbomam/deep-learning-ontology#11]
use `https://w3id.org/dlo/`

**Comment by turbomam** [comment]
might be more of a create operation than an update operation

**Comment by turbomam** [comment]
see also?
- https://github.com/berkeleybop/artificial-intelligence-ontology/issues/24

**Comment by turbomam** [comment]
I can't remember what DLO was, but it looks like there has been some progress on setting the AIO base URI to https://w3id.org/aio/ [here](https://github.com/berkeleybop/artificial-intelligence-ontology/blob/ca3c79a8abe4ea5217df4fb48325da2baed435ef/src/ontology/aio.Makefile#L24-L26)

But the prefixes and expansions in https://github.com/berkeleybop/artificial-intelligence-ontology/blob/main/src/ontology/aio-edit.owl are inconsistent

**Comment by caufieldjh** [comment]
you're right - the name of this changed from DLO -> artificial-intelligence-ontology (but also AIO) -> AIO
